GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -11,6 +11,7 @@ namespace gdjs {
     _debugDraw: PIXI.Graphics | null = null;
     _debugDrawContainer: PIXI.Container | null = null;
     _profilerText: PIXI.Text | null = null;
+    _showCursor: boolean = false;
```

Comment:
Should we call this

```suggestion
    _showCursorAtNextRender: boolean = false;
```

?